src/BusinessLayer/InternetReporting.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <QObject>
+#include <initializer_list>
+
+class BatteryData;
+class BatteryFaultsData;
+class CmuData;
+class DriverControlsData;
+class KeyMotorData;
+class LightsData;
+class MotorDetailsData;
+class MotorFaultsData;
+class MpptData;
+class QIODevice;
+class CommunicationService;
+
+class InternetReporting : public QObject

```

Comment:
If you're going to create an InternetReporitng, you need to change the name of the other service accordingly as well. `SerialReporting`.